webhooks/common.go

```diff
@@ -85,10 +85,37 @@ func NewFeatureFlagSourceSpec(env types.EnvConfig) *api.FeatureFlagSourceSpec {
 	}
 }
 
+func NewFeatureFlagInProcessConfigurationSpec() *api.FeatureFlagInProcessConfigurationSpec {
+	return &api.FeatureFlagInProcessConfigurationSpec{
+		Port:                  apicommon.DefaultRPCPort,
```

Comment:
```suggestion
		Port:                  apicommon.DefaultInProcessPort,
```

This basically doesn't allow us to correctly change the port, I think you used the wrong var

Reply:
good catch, should we allow configuration for all in-process parameters via env variables (and therefore also helm chart values)?

Reply:
in the end I adapted it for all configurable parameters, if it's not needed, I will remove it